docs/Manual/Deployment/Kubernetes/DeploymentResource.md

```diff
@@ -352,6 +352,14 @@ for `spec.mode: Single` and `2` for `spec.mode: ActiveFailover`).
 For the `syncworkers` group, it is highly recommended to use the same number
 as for the `dbservers` group.
 
+### `spec.<group>.minCount: number`
+
+Specifies a minimum for the count of servers. If set, a specification is invalid if `count < minCount`.
```

Comment:
What about `minCount <= maxCount`?

Reply:
This is implicit, since there is no `count` that could possible satisfy both: 
if there is `c` with `a <= c` and `c <= b` then `a <= b`.